js/main.js

```diff
@@ -8,3 +8,7 @@ function displayViewport(event) {
 window.addEventListener('load', displayViewport, false);
 window.addEventListener('resize', displayViewport, false);
 window.addEventListener('orientationchange', displayViewport, false);
+
+document.querySelectorAll('.Footer-toggle')[0].addEventListener('click', (event) => {
```

Comment:
Since this repo doesn't make use of Babel for transpiling, the arrow function won't work in IE (and will throw, causing the entire page to fail). You can replace it with a function declaration instead:

```suggestion
document.querySelectorAll('.Footer-toggle')[0].addEventListener('click', function(event) {
```